client/me/purchases/remove-purchase/index.jsx

```diff
@@ -130,7 +131,7 @@ const RemovePurchase = React.createClass( {
 				notices.success(
 					this.translate( '%(productName)s was removed from {{siteName/}}.', {
 						args: { productName },
-						components: { siteName: <em>{ selectedSite.slug }</em> }
+						components: { siteName: <em>{ slugToUrl( selectedSite.slug ) }</em> }
```

Comment:
This function is doing exactly the same as the [`domain` attribute](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/selectors.js#L247) of the site entity, which is what should be used instead.

`selectedSite.domain` instead of `slugToUrl( selectedSite.slug )`.